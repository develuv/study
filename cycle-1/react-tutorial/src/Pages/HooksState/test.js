import React, { useState, useEffect } from 'react';

function Example() {
  const [count, setCount] = useState(0);

  // componentDidMount, componentDidUpdate와 같은 방식으로
  useEffect(() => {
    // 브라우저 API를 이용하여 문서 타이틀을 업데이트합니다.
    document.title = `You clicked ${count} times`;
  });

  return (
    <div>
      <p>You clicked {count} times</p>
      <button onClick={() => setCount(count + 1)}>
        Click me
      </button>
    </div>
  );
}

export default Example;