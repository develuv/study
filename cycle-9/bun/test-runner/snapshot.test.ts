import { test, expect, jest } from "bun:test";

test("snapshot", async () => {
  console.log(jest)
  expect({ a: 3 }).toMatchSnapshot();
});