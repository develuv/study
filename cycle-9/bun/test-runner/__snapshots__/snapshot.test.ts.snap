// Bun Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
{
  "a": 3,
}
`;
